addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientDetailsEdit.js

```diff
@@ -25,7 +25,7 @@ odoo.define('point_of_sale.ClientDetailsEdit', function(require) {
             if (this.changes.image_1920) {
                 return this.changes.image_1920;
             } else if (partner.id) {
-                return `/web/image?model=res.partner&id=${partner.id}&field=image_128&write_date=${partner.write_date}&unique=1`;
+                return `/web/image?model=res.partner&id=${partner.id}&field=avatar_128&write_date=${partner.write_date}&unique=1`;
```

Comment:
I don't know the POS code very well but I think you need to adapt the other occurrences above too.

They use caching to allow offline usage, and the cache should be updated with the new avatar too if that is the one we want to display.

Reply:
The upper usages are used only when you change a customer's image in POS. Since we're updating the image, it should be written on the image_1920 field.